lib/routes/api/accounts.js

```diff
@@ -162,6 +162,40 @@ router.get('/:sid/RegisteredSipUsers', async(req, res) => {
   }
 });
 
+router.post('/:sid/RegisteredSipUsers', async(req, res) => {
+  const {logger, registrar} = req.app.locals;
+  const users = req.body;
+  if (!users && !Array.isArray(users)) {
```

Comment:
At the time created this PR, I would to avoid scan all keys if not required. If we want to get whole users, I may want to wait until @ajukes PR merged to redis helper lib

Reply:
Yes, but I would not want to change the behavior after we release this feature.  So I guess we will wait until I finish review and merge the other PR

Reply:
I have created new PR for registra to support this feature after the other PR is already merged. Please release that to new version and I will update this PR after that soon.

Reply:
jambonz-mw-registrar@0.2.6 has that change